Jenkinsfile-pr

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env groovy
+
+def lib = evaluate readTrusted('./jenkins.groovy')
+
+pipeline {
+    agent {
+        node {
+            label 'strimzi-pr'
+        }
+    }
+    parameters {
+        //execute all tests as default for each new Pull request
+        string(name: 'JUNIT_TAG', defaultValue: '', description: 'maven parameter for executing specific tests')
+    }
+    options {
+        timeout(time: 7, unit: 'HOURS')
+        ansiColor('xterm')
+    }
+    environment {
+        DOCKER_ORG="strimzi"
+        DOCKER_REGISTRY="docker.io"
+        DOCKER_TAG="latest"
+        MVN_ARGS="-DskipTests"  // Remove this when unit tests will be stabilized TODO jstejska
+        OPENSHIFT_URL="https://github.com/openshift/origin/releases/download/v3.10.0/openshift-origin-client-tools-v3.10.0-dd10d17-linux-64bit.tar.gz"
```

Comment:
Is it possible to use OC version v3.11.0?

Reply:
Yes, are you sure that all test suite is working on 3.11 ?

Reply:
I'm not sure, but I guess we need to support the latest stable version. And also if we have some problems for 3.11 we have to know about it.